### PR TITLE
Adding Intel 471 Connector v2

### PR DESCRIPTION
**Pull request by mmolenda**
### Proposed changes

* Introduce new Reports feed in place of IoC feed
* Add configuration item for Indicator decay score
* Improve Vulnerabilities mapper
* Map Intel 471 [General Intelligence Requirements (GIRs)](https://github.com/intel471/CU-GIR) into OpenCTI labels

### Related issues
* #3137

- [x] I consider the submitted work as finished
- [x] I tested the code for its functionality using different use cases
- [x] I added/update the relevant documentation (either on github or on notion)
- [x] Where necessary I refactored code to improve the overall quality

### CODE REVIEW

File: external-import/intel471_v2/entrypoint.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+# Correct working directory
+cd /opt/opencti-intel471
```
Comment (helene-nguyen) [inline external-import/intel471_v2/entrypoint.sh]: @mmolenda Could you change it and add v2 please to not having problem with v1 of connector ?

File: external-import/intel471_v2/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+version: '3'
+services:
+  connector-intel471:
+    image: opencti/connector-intel471:6.4.2
```
Comment (helene-nguyen) [inline external-import/intel471_v2/docker-compose.yml]: @mmolenda Could you change it and add v2 please to not having problem with v1 of connector ?

File: external-import/intel471_v2/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,16 @@
+FROM python:3.12-alpine
+ENV CONNECTOR_TYPE=EXTERNAL_IMPORT
+
+COPY src /opt/opencti-intel471
```
Comment (helene-nguyen) [inline external-import/intel471_v2/Dockerfile]: @mmolenda Could you change it and add v2 please to not having problem with v1 of connector ?

File: external-import/intel471_v2/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+# hadolint ignore=DL3003
+RUN apk update && apk upgrade && \
+    apk --no-cache add git build-base libmagic libxslt libxslt-dev libxml2 libxml2-dev && \
+    cd /opt/opencti-intel471 && \
```
Comment (helene-nguyen) [inline external-import/intel471_v2/Dockerfile]: Could you change it and add v2 please to not having problem with v1 of connector ?

**Review by Megafredo — APPROVED** [review]
hi @mmolenda,
It's ok for me, thank you for your contributions !

**Comment by mmolenda** [comment]
Hi @Megafredo, I think it will be cleaner to create a new branch/PR with signed commits. Are you ok with that?

**Comment by Megafredo** [comment]
Hi @mmolenda, yes no problem ;)
PS: Just don't forget to make the changes !

**Comment by mmolenda** [comment]
Opened #3146 with signed commits instead of this PR.

**Pull request closed without merging**